### ISSUE TITLE
Can't use Chinese in v2.2.0

### ISSUE THREAD

**Issue by NotAzay** [fyne-io/fyne#3056]
### Describe the bug:

Can't use Chinese in v2.2.0

### To Reproduce:
Just switch the version.

### Screenshots:

V2.2.0:
![2-2](https://user-images.githubusercontent.com/104895344/173035488-a502067b-bd4a-4995-9f51-686545a304e0.PNG)

V2.1.4:
![2-1-4](https://user-images.githubusercontent.com/104895344/173035537-a1158358-28e8-47ce-87f3-fa62d276358e.PNG)

### Example code:

```go
package main

import (
	"os"
	"strings"

	"fyne.io/fyne/v2/app"
	"fyne.io/fyne/v2/container"
	"fyne.io/fyne/v2/layout"
	"fyne.io/fyne/v2/widget"

	"github.com/flopp/go-findfont"
)

func init() {
	//设置中文环境
	fontPaths := findfont.List()
	for _, path := range fontPaths {
		// fmt.Println(path)
		//楷体:simkai.ttf
		//黑体:simhei.ttf
		if strings.Contains(path, "simkai.ttf") {
			os.Setenv("FYNE_FONT", path)
			break
		}
	}
}

func main() {
	mainGUI := app.New()
	mainWindow := mainGUI.NewWindow("中文测试")
	testLabel1 := widget.NewLabel("一二三四五，上山打老虎")
	testLabel2 := widget.NewLabel("老虎打不到，打到小松鼠")
	testContainer := container.New(layout.NewVBoxLayout(), testLabel1, testLabel2)
	mainWindow.SetContent(testContainer)
	mainWindow.ShowAndRun()
}

```

### Device (please complete the following information):
 - **OS:** Win10 Professional Edition
 - **Version:** 
 - **Go version:** 1.18.3
 - **Fyne version:** 2.2.0

**Comment by andydotxyz** [comment]
Confirmed. This is a regression.
The functionality is still there, as setting FYNE_FONT on the environment before launch works OK.
Need to look into how/why this changed and whether we want to support this going forward.

Of not is that setting up a custom theme with this font will work, that is a more robust approach as well https://developer.fyne.io/extend/custom-theme.

**Comment by andydotxyz** [comment]
Fixed for v2.2.1 release
